microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -134,7 +134,7 @@ Use the **Box Model** interactive diagram in the **Styles** panel to change the
 
    Depending on the size of your DevTools window, you may need to scroll to the bottom of the **Styles** panel to display the **Box Model**.
 
-1. Double-click the left margin in the **Box Model**, which currently has a value of `-`, meaning that the element doesn't have a `margin-left`.
+1. Double-click the left margin in the **Box Model**, which currently has a value of `-`. The `-` means that the element doesn't have a value for `margin-left`.
```

Comment:
add red boxes on the next couple screenshots

Reply:
added red boxes, should be visible now